cmake/projects/CLI11/hunter.cmake

```diff
@@ -54,6 +54,29 @@ hunter_add_version(
     4d900c543cc73b80bc31da20998b2d516d67e0ae
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    CLI11
+    VERSION
+    1.6.2
+    URL
+    "https://github.com/CLIUtils/CLI11/archive/v1.6.2.tar.gz"
+    SHA1
+    e5e91f4316b51c62b794e3c1575dd1309affae57
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    CLI11
+    VERSION
+    1.7.0
+    URL
+    "https://github.com/CLIUtils/CLI11/archive/v1.7.0.tar.gz"
+    SHA1
+    5e1daa3ee03d17e040242d77c53e80747e03c220
```

Comment:
SHA1 is `223a3c3dff2ccbb37e9caafb3f6308fc32564060`

Reply:
For some reason curl followed by sshsum was giving the wrong values on my Mac. Reran the downloads and checks using wget on Linux and that worked.